src/Microsoft.Extensions.ProcessHelper.Sources/project.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "version": "1.0.0-*",
+  "shared": "*.cs",
+  "dependencies": {
+  },
+  "frameworks": {
+    "net451": { },
+    "netstandard1.4": {
+      "dependencies": {
+        "System.Diagnostics.Process": "4.1.0-*"
```

Comment:
Shouldn't need this. Probably adding a reference to `System.Runtime` should suffice.